tests/containers_test.toit

```diff
@@ -7,7 +7,8 @@ import expect show *
 
 main:
   images/List := containers.images
-  expect_list_equals [containers.current] images
+  expect_equals 2 images.size
```

Comment:
Not really sure this is the behavior I want, but for now we do return the system image in the list of containers.